db/migrate/20200429005140_add_dash_underscore_typo_detection_index_to_rubygems.rb

```diff
@@ -0,0 +1,21 @@
+#
+# Create a PostgreSQL functional index on all Gem's names with - and _ charcters removed.
+# This index is used by GemTypo to rapidly check newly uploaded Gems for matches against
+# other Gems that are variations of another already published Gem simply by the variation in
+# - and _ characters.
+#
+# This simple rule would have stopped the vast majority of typo squat attacks observed in
+# 2019 and 2020 according to an analysis of the SQL database.
```

Comment:
The first paragraph is probably fine, not so sure if we need lines 7 and 8.

Reply:
This comment has been updated to remove the second paragraph.